Crash when switching the app to split screen mode

### Steps to reproduce

- Open the app 
- open any surah
- use the system split screen feature and select another app
- app crashes 

### Expected behavior

Split screen should work as usual when switched to split screen mode.

### Actual behavior

App crashed when i selected the second app. The app works fine in main screen. It only crashes when switched from the surah / ayah view

### QuranApp version

2025.12.07.1

### Android version

Android 13

### Other details

Crash logs

java.lang.RuntimeException: Unable to pause activity {com.quranapp.android/com.quranapp.android.activities.ActivityReader}: java.lang.NullPointerException: Attempt to invoke virtual method 'int androidx.recyclerview.widget.LinearLayoutManager.N0()' on a null object reference
	at android.app.ActivityThread.performPauseActivityIfNeeded(ActivityThread.java:5201)
	at android.app.ActivityThread.performPauseActivity(ActivityThread.java:5152)
	at android.app.ActivityThread.handlePauseActivity(ActivityThread.java:5104)
	at android.app.servertransaction.PauseActivityItem.execute(PauseActivityItem.java:47)
	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2291)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:210)
	at android.os.Looper.loop(Looper.java:299)
	at android.app.ActivityThread.main(ActivityThread.java:8319)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:556)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1038)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'int androidx.recyclerview.widget.LinearLayoutManager.N0()' on a null object reference
	at com.quranapp.android.activities.ActivityReader.g0(SourceFile:3)
	at com.quranapp.android.activities.ActivityReader.onPause(SourceFile:2)
	at android.app.Activity.performPause(Activity.java:8374)
	at android.app.Instrumentation.callActivityOnPause(Instrumentation.java:1531)
	at android.app.ActivityThread.performPauseActivityIfNeeded(ActivityThread.java:5191)
	... 14 more


### Acknowledgements

- [x] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [x] I have written a short but informative title.

--- a/app/src/main/java/com/quranapp/android/activities/ActivityReader.java
+++ b/app/src/main/java/com/quranapp/android/activities/ActivityReader.java
@@ -1383,6 +1383,10 @@ public class ActivityReader extends ReaderPossessingActivity implements SmoothAu
 
     private void saveReaderState(boolean saveToHistory) {
         // Get first & last visible item positions (both could be same)
+        if (mLayoutManager == null) {
+            return;
+        }
+
         int firstPos = mLayoutManager.findFirstVisibleItemPosition();
         int lastPos = mLayoutManager.findLastVisibleItemPosition();
 

